src/He.PipelineAssessment.UI/Features/Error/ErrorController.cs

```diff
@@ -23,6 +26,8 @@ public IActionResult Index()
 
             var errorMessage = $"An error occurred while processing your request {exception.Message}";
 
+            _logger.LogError(errorMessage);
```

Comment:
Im not sure if logging just the error message is the best approach here - 
The reason being we would be missing out on a lot of additional information (Such as The Stack Trace, Inner Exception etc)
It may be worth logging the error entirely - i.e. [logger.LogError(exception, "Error while processing request from {Address}", address)](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.logerror?view=net-9.0-pp)